continuous_integration/install.sh

```diff
@@ -21,9 +21,12 @@ if [[ "$DISTRIB" == "conda" ]]; then
 
     # Use the miniconda installer for faster download / install of conda
     # itself
+    mkdir -p download
+    cd download
     wget http://repo.continuum.io/miniconda/Miniconda-3.6.0-Linux-x86_64.sh \
```

Comment:
I am not Travis expert, but how would wget know that the file may exist in the cache?